tracer/src/Datadog.AutoInstrumentation.NativeLoader/dynamic_dispatcher.cpp

```diff
@@ -1,6 +1,13 @@
 #include "dynamic_dispatcher.h"
 
+#ifdef LINUX
+#include "../../../shared/src/native-src/filesystem.hpp"
+namespace fs = ghc::filesystem;
+#else
 #include <filesystem>
+namespace fs = std::filesystem;
+#endif
```

Comment:
This is a total nitpick, so feel free to resolve without doing, but maybe consider extracting this to a `fs.h` file to avoid the code duplication?

Reply:
you are right.
will do

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2527/commits/09f07d7f541e60bc34cea8fb3cf8db3d5649a065